operate/schema/src/main/java/io/camunda/operate/store/opensearch/client/sync/OpenSearchBatchOperations.java

```diff
@@ -71,6 +85,64 @@ private void processBulkRequest(OpenSearchClient osClient, BulkRequest bulkReque
     }
   }
 
+  private BulkRequest validateIndices(BulkRequest bulkRequest, boolean ignoreNullIndex)
```

Comment:
Please consider moving some of these functions into helper classes with unit tests so we avoid having a large monolithic class